lib/app/features/chat/messages/views/components/messaging_bottom_bar/components/more_content_view.dart

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/material.dart';
+import 'package:ion/app/extensions/extensions.dart';
+import 'package:ion/generated/assets.gen.dart';
+
+final double moreContentHeight = 206.0.s;
+
+class MoreContentView extends StatelessWidget {
+  const MoreContentView({super.key});
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      height: moreContentHeight,
+      width: double.infinity,
+      color: context.theme.appColors.secondaryBackground,
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceAround,
+            children: [
+              Column(
```

Comment:
Extract each `Column` to a separate widget to improve readability.

Reply:
Or, as I can see, they are similiar, so we need only 1 widget with  Icon and Label params.